crates/ra_syntax/src/ast/traits.rs

```diff
@@ -136,7 +136,18 @@ pub trait DocCommentsOwner: AstNode {
                         prefix_len
                     };
 
-                line[pos..].to_owned()
+                let end = if comment.kind().shape.is_block() && line.ends_with("*/") {
+                    // FIXME: Use `nth_back` here once stable
+                    if line.chars().rev().nth(2).map(|c| c.is_whitespace()).unwrap_or(false) {
```

Comment:
Let's check that `c` s is ascii as well, otherwise this will panic on weird uicode whitespace characters.

Reply:
Maybe use `line[..line.len() - 2].trim_end()` below instead.

Reply:
I like the `trim_end()` on every comment line. Is there a reason we can't unconditionally do that?

Reply:
I just did that in the latest commit.